data/json/recipes/recipe_others.json

```diff
@@ -648,20 +648,14 @@
   "skill_used": "survival",
   "time": 5000,
   "autolearn": true,
+  "using" : [ [ "cordage_short", 2 ], [ "filament", 40 ] ],
```

Comment:
If i'm not mistaking, this still means AND and not OR. You don't want to use "using" you should keep them down in "components".

Reply:
No, this means OR.